tests/bsim/bluetooth/host/privacy/peripheral/nrf5340_cpunet-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,24 @@
+CONFIG_IPC_SERVICE=y
+CONFIG_MBOX=y
+CONFIG_ISR_STACK_SIZE=1024
```

Comment:
Just as info for the future so you don't feel the need to carry this around: 
In the POSIX arch the stacks only need to be CONFIG_ARCH_POSIX_RECOMMENDED_STACK_SIZE, beyond that they are unused. ISR_TABLES_LOCAL_DECLARATION is a nop, and CBPRINTF_REDUCED_INTEGRAL is kinda irrelevant. (None of this harms mind you)